qtawesome/icon_browser.py

```diff
@@ -260,6 +311,31 @@ def data(self, index, role):
         return super().data(index, role)
 
 
+class ToolBarGroup(QtWidgets.QWidget):
```

Comment:
Regarding this custom widget to add titles, what do you think if instead we just add tooltips to the relevant widgets so people can check/understand what they do?

So for example for the line edit that filters the icons:

![filter](https://github.com/spyder-ide/qtawesome/assets/16781833/af6c1f62-d65e-492e-b3d5-f8ed8391b11a)

For the text for each element, it could be something like:

* Font combobox: `Select fonts prefix which icons will be included in the filtering`
* Filter line edit: `Filter icons by name`
* Selected icon line edit: `Full identifier of the currently selected icon`
* Copy button: `Copy selected icon full identifier to the clipboard`
* Theme combobox: `Select color palette for the icons and the icon browser`
* Columns combobox: `Select number of columns the icons list is showing`